content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -102,22 +102,30 @@ The `Peppol <https://peppol.org/about/>`_ network ensures the exchange of docume
 between enterprises and governmental authorities. It is primarily used for electronic invoicing, and
 its access points (connectors to the Peppol network) allow enterprises to exchange electronic
 documents.
-Odoo is now an **access point** enabling electronic invoicing transactions without the need to send
-invoices and bills by email or post.
+Odoo is now an **access point** and an :abbr:`SMP (Service Metadata Publisher)`,
+enabling electronic invoicing transactions without the need to send invoices and
+bills by email or post.
 
 Configuration
 -------------
 
-First, :ref:`install <general/install>` the :guilabel:`Peppol` module (`account_peppol`).
+:ref:`Install <general/install>` the :guilabel:`Peppol` module (`account_peppol`).
 
 .. image:: electronic_invoicing/peppol-module.png
    :alt: Peppol module install
 
-Then, go to :menuselection:`Accounting --> Configuration --> Settings`, tick the
-:guilabel:`Use PEPPOL Invoicing`, and fill in the following information:
+Registration
+~~~~~~~~~~~~
 
-- `PEPPOL EAS <https://ec.europa.eu/digital-building-blocks/wikis/display/DIGITAL/Code+lists/>`_
-- :guilabel:`Peppol Endpoint`
+Go to :menuselection:`Accounting --> Configuration --> Settings`, tick the
+:guilabel:`Use PEPPOL Invoicing` checkbox, and fill in the following information:
+
+- `Peppol EAS <https://ec.europa.eu/digital-building-blocks/wikis/display/DIGITAL/Code+lists/>`_.
+  This is the Peppol Electronic Address Scheme and usually depends on your company's country.
+  Odoo often prefills the most commonly used EAS code in your country.
+  For example, the preferred EAS code for most companies in Belgium is 0208.
+- `Peppol Endpoint <https://docs.peppol.eu/edelivery/codelists/v8.7/Peppol%20Code%20Lists%20-%20Participant%20identifier%20schemes%20v8.7.html>`_.
```

Comment:
@dbkosky @aliyatastemirova 
Let's assume we won't update the links. At some point, this will be deprecated. Can't we use this link instead and mention to open the participant identifiers? :) https://docs.peppol.eu/edelivery/codelists/

Reply:
yes, that sounds good to me :slightly_smiling_face: